src/xrpld/app/tx/detail/Clawback.cpp

```diff
@@ -39,7 +39,7 @@ preflightHelper<Issue>(PreflightContext const& ctx)
     if (ctx.tx.isFieldPresent(sfHolder))
         return temMALFORMED;
 
-    AccountID const issuer = ctx.tx[sfAccount];
+    AccountID const issuer = ctx.account;
```

Comment:
Should be const&.

Reply:
since it only used once, I'll remove this line and simplify to
```
    if (ctx.account  == holder || isXRP(clawAmount) || clawAmount <= beast::zero)
        return temBAD_AMOUNT;
```